apps/dashboard/app/(app)/logs/components/filters/components/search-combobox/search-combobox.tsx

```
@@ -0,0 +1,167 @@
+  const handleEditKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === KEYS.ENTER) {
+      setCurrentFocusedItemIndex(NO_ITEM_EDITING);
+    }
+  };
+
+  return (
+    <Popover open={open} onOpenChange={setOpen}>
+      <PopoverTrigger asChild>
+        <Button
+          variant="outline"
+          role="combobox"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<select>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)